### PR TITLE
Allow Kotlin-only activities

### PR DESCRIPTION
**Pull request by colleenmcginnis**
In the past, all Android docs had _at least_ Java code snippets and examples so we required the `java` prop in `ContextlessAndroidActivityToggle`. In most future releases, all Android docs will have Kotlin snippets (and will likely _not_ have Java snippets). 

## How to test

A copy of this component is used on the `cmcg-adding-nav-1.0-examples` branch in /android-docs.

## QA checklist

- [ ] No errors logged to console.
- [ ] Accessibility score in [Chrome DevTools Audit/Lighthouse](https://developers.google.com/web/tools/lighthouse#devtools) is 100% with Lighthouse version: `#.#.#`.
- [ ] Component is accessible at mobile-size viewport.
- [ ] Component is accessible at tablet-size viewport.
- [ ] Component is accessible at laptop-size viewport.
- [ ] Component is accessible at big-monitor-size viewport.
- [ ] Create a PR in a site repo, copy the component, and test it. Push to staging and let the reviewer know they can also test the component there.

Open the test cases app locally on:

- [ ] Chrome, no errors logged to console.
- [ ] Firefox, no errors logged to console.
- [ ] Safari, no errors logged to console.
- [ ] Edge, no errors logged to console.
- [ ] IE11, no errors logged to console.
- [ ] Mobile Safari, no errors logged to console.
- [ ] Android Chrome, no errors logged to console.

### CODE REVIEW

File: src/components/contextless-android-activity-toggle/contextless-android-activity-toggle.js [hunk 1]
```diff
@@ -23,7 +23,19 @@ export default class ContextlessAndroidActivityToggle extends React.Component {
...
+      } else if (java === undefined) {
+        /* If there is no java code, use kotlin. */
+        selectedCode = kotlin;
+      } else if (this.checkPreference('kotlin')) {
+        /** If there is both java and kotlin code,
+         * use the preferred language. */
+        selectedCode = kotlin;
+      } else {
+        selectedCode = java;
+      }
```
Comment (katydecorah) [inline src/components/contextless-android-activity-toggle/contextless-android-activity-toggle.js:38]: @colleenmcginnis you may want to consider breaking this out into a function that you can test. Nested logic can be hard to parse and having test cases can help debug and future problems (if any). If you'd like to give this a try, here are some suggestions on how to approach it:

1. Above `export default class ContextlessAndroidActivityToggle..` you could create a new function, something like `export function findSelectedCode(kotlin, java, preference) {}` (this is passing the kotlin code, java code, and the user's preference).
2. The interior of the new function could be lines 24 + 26 - 38, and the function would return the value of `selectedCode`. The only other change you'd want to make is at line 32, where you could evaluate `preference === 'kotlin'` instead of invoking `this.checkPreference` since we are now outside of the class and cannot reach that function.
3. Coming back to line 26, we can call that function: `selectedCode = findSelectedCode(kotlin, java, context.preferredLanguage.android)`
4. Create a new test file in the component's `__tests__` dir, import the function `import {findSelectedCode} from '../contextless-android-activity-toggle';` and build out a series of test cases to make sure it returns the selected code. Example: `expect(findSelectedCode('kotlin code', 'java code', 'kotlin')).toEqual("kotlin code");`

**Comment by colleenmcginnis** [comment]
@katydecorah I attempted to implement your suggestion in https://github.com/mapbox/dr-ui/pull/278#discussion_r427301422! Can you take another look?
